redaxo/src/addons/media_manager/lib/effects/effect_convert2img.php

```diff
@@ -177,4 +231,32 @@ private function getConvertPath()
         }
         return $path;
     }
+
+    /**
+     * @package redaxo\media-manager
+     * return bool
+     */
+    private function isVideoToImageConversionSupported(): bool
+    {
+
+        $inputFile = $this->media->getMediaPath();
+        $inputExt = pathinfo($inputFile, PATHINFO_EXTENSION);
+
+        if ($this->isFfmpegAvailable()) {
+            return in_array($inputExt, self::VIDEO_TO_IMAGE_TYPES);
+        }
+        return false;
+
+    }
+
+    private function isFfmpegAvailable(): bool
+    {
+        $ffmpegPath = 'ffmpeg'; // change to full path if necessary
+        $output = [];
+        $returnVar = -1;
+
+        exec($ffmpegPath . ' -version', $output, $returnVar);
```

Comment:
Es sollte auch geprüft werden, ob die Funktion `exec` überhaupt verfügbar ist, bevor sie aufgerufen wird. Siehe in `getConvertPath` in dieser Datei.

Reply:
exec wird nun geprüft